VDA-Verification-Base/contracts/VDATrustedSignerManager.sol

```diff
@@ -0,0 +1,79 @@
+//SPDX-License-Identifier: MIT
```

Comment:
Can you explain the reason for this refactor?

Reply:
I just separated the `VDATrustedSignerManager` from original `VDAVerificationBase` contract.
In some cases, we need a contract that includes feature for trusted signers.
For example, in the `claimXPReward()` function of  `XPReward` contract, we didn't add the `requestSignature` and `requestProof` parameters at first. Here, we need to manage the trusted signers. So I separated the `VDATrustedSignerManager` and made the `XPReward` contract inherits that.
Later, we added the `requestSignature` and `requestProof`. So I made the `XPReward` contract inherits the `VDAVerificationBase`.

By the way, we can create some contracts that inherits the `VDATrustedSignerManager` only later.

Reply:
Ok, noted